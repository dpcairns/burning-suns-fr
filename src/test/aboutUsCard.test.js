import React from 'react';
import { shallow } from 'enzyme';
import Test from '../aboutUsCard';
import kPhoto from '../kigers.jpg';


describe('First React component test with Enzyme', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<Test name={'me'}
        alt={'space the finial frontear'}
        photo={kPhoto}
        detail={`Lorem ipsum dolor sit amet, 
        consectetur adipiscing elit, sed do eiusmod
         tempor incididunt ut labore et dolore magna
         aliqua. In hendrerit gravida rutrum quisque non
          tellus orci ac. Lorem mollis aliquam ut porttitor
          leo a diam sollicitudin tempor. Vitae congue mauris
           rhoncus aenean vel elit scelerisque mauris. Leo duis u
           t diam quam. Vitae nunc sed velit dignissim sodales ut
           eu sem. A pellentesque sit amet porttitor eget dolor morbi
            non.`} />);
      expect(wrapper).toMatchSnapshot()
     });
 });