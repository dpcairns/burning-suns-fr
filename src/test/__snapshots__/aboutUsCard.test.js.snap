// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<div>
  <li
    className="AboutUsStyle"
  >
    <div
      className="immageContainer"
    >
      <h2>
        me
      </h2>
      <div
        className="immageContatin"
      >
        <img
          alt="space the finial frontear"
          src="kigers.jpg"
        />
      </div>
      <WithStyles(ForwardRef(Typography))>
        Lorem ipsum dolor sit amet, 
        consectetur adipiscing elit, sed do eiusmod
         tempor incididunt ut labore et dolore magna
         aliqua. In hendrerit gravida rutrum quisque non
          tellus orci ac. Lorem mollis aliquam ut porttitor
          leo a diam sollicitudin tempor. Vitae congue mauris
           rhoncus aenean vel elit scelerisque mauris. Leo duis u
           t diam quam. Vitae nunc sed velit dignissim sodales ut
           eu sem. A pellentesque sit amet porttitor eget dolor morbi
            non.
      </WithStyles(ForwardRef(Typography))>
    </div>
  </li>
</div>
`;
